pkgs/development/python-modules/pijuice/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, fetchFromGitHub, buildPythonPackage, pythonOlder,
+smbus-cffi, urwid }:
+
+buildPythonPackage rec {
+  pname = "pijuice";
+  version = "1.7";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "PiSupply";
+    repo = "PiJuice";
+    rev = "3ba6719ab614a3dc7495d5d9c900dd4ea977c7e3";
```

Comment:
Why aren't we using the tag?

Reply:
Unfortunately, this project doesn't tag their releases.  I had to look at which commit altered the `__version__` field in `pijuice.py`.

Reply:
```suggestion
    # Unfortunately project doesn't tag releases. This commit altered the __version__ field
    rev = "3ba6719ab614a3dc7495d5d9c900dd4ea977c7e3";
```

Reply:
Comment added.